webapp/src/components/common/FormTable/Table.tsx

```diff
@@ -48,7 +48,7 @@ function Table(props: TableProps) {
   const handleAfterChange: HandsontableProps["afterChange"] =
     function afterChange(this: unknown, changes, ...rest): void {
       changes?.forEach(([row, column, _, nextValue]) => {
-        setValue(`${data[row].id}.${column}`, nextValue);
+        setValue(`${data[row].id}.${column}`, nextValue ?? "");
```

Comment:
No. Like I said, `nextValue` is not necessary a string.

Reply:
`null` value is possible only on strings. when you press delete on booleans `true` or `false` only is possible. on numbers inputs its disabled.

So I dont see in what case we can have an other type that is `null` ?

Reply:
We can maybe check for the type of `nextValue` ? 
```javascript
if(typeof nextValue === 'string' && nextValue === null) {
...
}
```

Reply:
We can check the type of nextValue and add a specific case for strings only ? 

```javascript
changes?.forEach(([row, column, _, nextValue]) => {
  if (typeof nextValue === 'string') {
    setValue(`${data[row].id}.${column}`, nextValue ?? "");
  } else {
    setValue(`${data[row].id}.${column}`, nextValue);
  }
});
```

Reply:
EDIT:

`afterChange` event in Handsontable provides the previous value of the cell `prevValue`, in addition to `nextValue`. We can leverage this to determine if the cell was originally a `string`. If it was, and if `nextValue` is `null`, we can then replace it with an empty string. This ensures that we only replace null with "" for cells that were originally strings.

```javascript
  const handleAfterChange: HandsontableProps["afterChange"] =
    function afterChange(this: unknown, changes, ...rest): void {
      changes?.forEach(([row, column, prevValue, nextValue]) => {
        if (typeof prevValue === "string" && nextValue === null) {
          setValue(`${data[row].id}.${column}`, "");
        } else {
          setValue(`${data[row].id}.${column}`, nextValue);
        }
      });
      restProps.afterChange?.call(this, changes, ...rest);
    };
```
Tested, it works as expected, I dont see a better solution if this does not suits you I let you provide a better one

Reply:
Use `beforeChange` instead.